sky/clouds/service_catalog/aws_catalog.py

```diff
@@ -35,17 +36,28 @@
                                 pull_frequency_hours=_PULL_FREQUENCY_HOURS)
 
 
-def _apply_az_mapping(df: 'pd.DataFrame') -> 'pd.DataFrame':
+def _apply_az_mapping(df: pd.DataFrame) -> pd.DataFrame:
     """Maps zone IDs (use1-az1) to zone names (us-east-1x).
 
-    Such mappings are account-specific and determined by AWS.
+    Such mappings are account-specific and determined by AWS. The mpaaing will
+    also remove the regions that are not supported by the user account. If the
+    user does not have AWS credentials configured, we use a default mapping.
```

Comment:
```suggestion
    Such mappings are account-specific and determined by AWS. We fetch the mappings from AWS, which requires AWS credentials. If the user does not have AWS credentials configured, we use a default mapping.
    
    The mappings will also serve to remove from '_df' the regions that are not supported by the user account. 
```
Actually, can we briefly explain what "default mapping" means, and why that is ok?

Reply:
Modified and added the reason for default mapping. Thanks!